handler/cantabular_metadata_export.go

```diff
@@ -219,7 +219,12 @@ func (h *CantabularMetadataExport) Handle(ctx context.Context, workerID int, msg
 }
 
 func (h *CantabularMetadataExport) exportTXTFile(ctx context.Context, e *event.CSVCreated, m dataset.Metadata, isPublished bool, isCustom bool) (*downloadInfo, error) {
-	b := text.NewMetadata(&m, isCustom, e.FilterOutputID, h.cfg.DownloadServiceURL)
+	var b []byte
+	if isCustom {
+		b = text.NewMetadataCustom(&m, e.FilterOutputID, h.cfg.DownloadServiceURL)
+	} else {
+		b = text.NewMetadata(&m, e.FilterOutputID, h.cfg.DownloadServiceURL)
+	}
```

Comment:
It seems we have other `New` functions that accept the `isCustom` boolean parameter, instead of offering a new custom function (I can see in this PR that at least csvw does it like that). I am not familiar with this repo but it'd be good to be consistent?